setup.py

```diff
@@ -49,8 +49,26 @@
     ]},
     install_requires=['pytz'],
     extras_require={
-        "bcrypt": ["bcrypt"],
-        "argon2": ["argon2-cffi >= 16.1.0"],
+        'admindocs': ['docutils'],
+        'argon2': ['argon2_cffi>=16.1.0'],
+        'bcrypt': ['bcrypt'],
+        'gis': [
+            'geoip2',
+            'numpy'
+        ],
+        'geoip': ['geoip2'],
+        'jinja2': ['Jinja2>=2.9.2'],
+        'memcached': ['python3-memcached'],
+        'numpy': ['numpy'],
+        'mysql': ['mysqlclient>=1.3.7'],
```

Comment:
`mysqlclient>=1.3.3`

https://docs.djangoproject.com/en/dev/ref/databases/#id6

Reply:
The test requirements specify >=1.3.7 (https://github.com/django/django/blob/master/tests/requirements/mysql.txt) "Due to a bug that will be fixed in mysqlclient 1.3.7."

I'm Not sure what's more correct.

Reply:
See https://github.com/django/django/pull/9472 for a documentation change request.